README.md

```diff
@@ -9,44 +9,81 @@ distributing many simulations across a computing cluster, and selectively
 mutating rare materials--in an iterative process.
 
 ## Getting started
+This package was built to be run locally, or on a computing cluster. To run on   
+a cluster, the following must be installed:   
+  - Torque/PBS (qsub)   
```

Comment:
technically, they can run it on any cluster regardless of whether they are using Torque / pbs. Though we only supply a sample script for qsub!